types/node/ts3.1/async_hooks.d.ts

```diff
@@ -115,6 +115,19 @@ declare module "async_hooks" {
          */
         constructor(type: string, triggerAsyncId?: number|AsyncResourceOptions);
 
+        /**
+         * Binds the given function to the current execution context.
+         * @param fn The function to bind to the current execution context.
+         * @param type An optional name to associate with the underlying `AsyncResource`.
+         */
+        static bind<Func = (...args: any[]) => any>(fn: Func, type?: string): Func & { asyncResource: AsyncResource };
```

Comment:
should we use something like `bind<Func extends Function = (...args: any[]) => any>...` to limit to functions?

Reply:
Make sense! I will change it.